product-service/model/productsModel.ts

```diff
@@ -1,5 +1,25 @@
-import { HttpError } from './HttpError';
-import productsList from './productsList.json';
+import { Client, ClientConfig } from 'pg';
+import { ProductsModelError } from './ProductsModelError';
+
+const {
+  PGUSER,
+  PGHOST,
+  PGDATABASE,
+  PGPASSWORD,
+  PGPORT,
+} = process.env;
+
+const dbOptions: ClientConfig = {
+  user: PGUSER,
+  host: PGHOST,
+  database: PGDATABASE,
+  password: PGPASSWORD,
+  port: Number(PGPORT),
+  ssl: {
+    rejectUnauthorized: false,
+  },
+  connectionTimeoutMillis: 5000,
+};
```

Comment:
@viacheslav-karnaukh I would strongly suggest to encapsulate db connection functionality in 1 module and export this module across your app. This way, if you need to change database user/host/password, you'll only need to change it in 1 file.

It's a wish for the future, no need to implement it right now, it won't affect your score.